# 1ª Questão

In [1]:
x = "42: The answer to life, the universe and everything"

In [2]:
# seu código aqui:
int(x[:2])/2


21.0

output: 21.0

Clique duas vezes para ver a resposta

<!--
int(x[:2])/2
-->

# 2ª Questão

In [1]:
lista1 = [1.2, 5]
lista2 = ['1']
lista3 = ['a']
lista4 = ['b']
lista5 = ['60', 60]

In [2]:
lista5[0]=int(lista5[0])+lista5[1]
print(lista5)

[120, 60]


In [7]:
lista=[lista1+lista2]+lista3+lista4+lista5
lista.remove(60)
print(lista)

[[1.2, 5, '1'], 'a', 'b', 120]


In [ ]:
## Outra forma de achar o elemento lista.index(elemento)

output: [[1.2, 5, '1'], 'a', 'b', 120]

Clique duas vezes para ver a resposta

<!--
lista5[0] = int(lista5[0]) + lista5[1]
lista = [lista1 + lista2] + lista3 + lista4 + lista5
del(lista[lista.index(60)])
lista
-->

# 3ª Questão

In [17]:
lista = [[1.2, 5, '1'], 'a', 'b', 180] 

In [18]:
# seu código aqui:
lista[0][2]=int(lista[0][2])
lista.reverse()
print(lista)

[180, 'b', 'a', [1.2, 5, 1]]


output: [180, 'b', 'a', [1.2, 5, 1]]

Clique duas vezes para ver a resposta
dropdown
<!--
lista[0][2] = int(lista[0][2])
lista.reverse()
lista
-->

# 4ª Questão

In [29]:
lista1 = [10,10,10,10,'a','b','c','d']
lista2 = ['f','g','h',20,20,20,'20.5']
lista = []

In [36]:
# seu código aqui: # porque e

lista = lista1[:4]+lista2[3:]
lista[-1]=float(lista[-1]) # menos 1 é o ultimo que estava como string
print(' A soma de todos os números é igual a {}'.format(sum(lista)))


 A soma de todos os números é igual a 120.5


output: A soma de todos os números é igual a 120.5

dropdown
Clique duas vezes para ver a resposta

<!--
lista = lista1[:4]+lista2[3:]
lista[-1] = float(lista[-1])
print('A soma de todos os números é igual a {}'.format(sum(lista)))
-->